app/redux/reducers/ConnectedDeviceReducer.js

```diff
@@ -45,7 +49,8 @@ const ConnectedDeviceReducer = ( state = defaultState, action) => {
             return Object.assign({}, state, {
                 status: 'DISCONNECTED',
                 deviceName: null,
-                deviceIdentifier: null
+                deviceIdentifier: null,
+                numDisconnects: state.numDisconnects += 1,
```

Comment:
This should only log disconnect count if it's a non manual disconnect.

Furthermore, should be +1 not +=1

Reply:
done